framework/common/logging.h

```diff
@@ -31,6 +31,8 @@
 #define __FILENAME__ (static_cast<const char *>(__FILE__) + ROOT_PATH_SIZE)
 
 #define LOGI(...) spdlog::info(__VA_ARGS__);
-#define LOGW(...) spdlog::warn(__VA_ARGS__);
+#define LOGW(...) void(0);
```

Comment:
did you mean to leave this in here?

Reply:
For now yes, helps clean the logs up whilst testing shader changes. This will be reverted when the PR is out of WIP

Will leave this comment unresolved as a reminder